support-frontend/assets/pages/paper-subscription-landing/components/hero/campaignHeaderStyles.js

```diff
@@ -0,0 +1,204 @@
+import { css } from '@emotion/core';
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { brand, brandAlt } from '@guardian/src-foundations/palette';
+import { headline, body } from '@guardian/src-foundations/typography';
+
+export const campaignHeader = css`
+  display: flex;
+  align-items: center;
+`;
+
+export const campaignCopy = css`
+  display: none;
+  position: relative;
+  z-index: 10;
+  margin-top: ${space[2]}px;
+
+  ${from.phablet} {
+    margin-top: 2rem;
+  }
+
+  ${from.desktop} {
+    margin-top: 3rem;
+  }
+
+  ${from.leftCol} {
+    margin-top: 5rem;
+  }
+
+  ${from.phablet} {
+    font-size: 35px;
+    line-height: 1;
+    display: block;
+    padding-left: ${space[1]}px;
+  }
+
+  ${from.tablet} {
+    padding-left: 0;
+  }
+`;
+
+export const heading = css`
+  ${headline.small({ fontWeight: 'bold' })};
+  color: ${brand[400]};
+  margin-bottom: ${space[3]}px;
+
+  ${from.desktop} {
+    ${headline.medium({ fontWeight: 'bold' })};
+  }
+
+  ${from.leftCol} {
+    ${headline.large({ fontWeight: 'bold' })};
+  }
+`;
+
+export const headingLineOne = css`
+  position: relative;
+  z-index: 5;
+`;
+
+export const headingLineTwo = css`
+  ${until.leftCol} {
+    display: block;
+    width: 400px;
+  }
+
+  ${until.desktop} {
```

Comment:
It's a bit confusing to me that there are two `until` mqs which overlap, I can't tell which one would win at tablet for instance.

Reply:
Yes! In fact this is a hang over from the previous version of the hero and these styles aren't needed anymore. I have deleted them.